addons/mass_mailing/models/mail_mail.py

```diff
@@ -95,13 +95,12 @@ def _send_prepare_values(self, partner=None):
         return res
 
     @api.multi
-    def _postprocess_sent_message(self, failure_type=None, **kwargs):
+    def _postprocess_sent_message(self, success_pids, failure_reason=False, failure_type=None):
         mail_sent = not failure_type  # we consider that a recipient error is a failure with mass mailling and show them as failed
         for mail in self:
             if mail.mailing_id:
                 if mail_sent is True and mail.statistics_ids:
                     mail.statistics_ids.write({'sent': fields.Datetime.now(), 'exception': False})
                 elif mail_sent is False and mail.statistics_ids:
                     mail.statistics_ids.write({'exception': fields.Datetime.now()})
-        return super(MailMail, self)._postprocess_sent_message(failure_type=failure_type, **kwargs)
-    
\ No newline at end of file
+        return  super(MailMail, self)._postprocess_sent_message(success_pids, failure_reason=failure_reason, failure_type=failure_type)
```

Comment:
LGTM I just prefer to keep the python2 super to keep a minimal diff. It is also a good practice in case of backport even if it is unlikely here. Can you please squash the commits, apply my change on super and force-push ?

Reply:
OK, I just changed to `super()` because anyways I was going to touch the line. Changed now.